core/src/main/java/io/neow3j/protocol/core/HexUtils.java

```diff
@@ -0,0 +1,31 @@
+package io.neow3j.protocol.core;
+
+import io.neow3j.crypto.KeyUtils;
+import io.neow3j.utils.ArrayUtils;
+import io.neow3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+public class HexUtils {
+
+    public static String reverse(String input) {
```

Comment:
This method should be moved to the `Numeric` class, even if it's not "numeric" per se. It's just to aggregate all the util methods related to hex, arrays, and others. Also, please, can you move the test to the `NumericTest` class? :-)